.github/workflows/ci.yml

```diff
@@ -0,0 +1,22 @@
+name: Continuous Integration 
+
+on:
+  pull_request:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
I think it would be worth testing against the latest released version as well as the latest pre-released version corresponding to the master branch of the repo.